lib/rubocop/cop/naming/method_name.rb

```diff
@@ -10,8 +10,8 @@ module Naming
       #
       #   Naming/MethodName:
       #     AllowedPatterns:
-      #       - '\A\s*onSelectionBulkChange\s*'
-      #       - '\A\s*onSelectionCleared\s*'
```

Comment:
Am I wrong about this? I looked at the code, and I can't see how the method name could ever have a space in it, and be intended to still be allowed.

We pass in

```ruby
def my_method_name
    ^^^^^^^^^^^^^^
    
attr_reader :my_attr_name
             ^^^^^^^^^^^^
end
```

so I don't see how spaces would appear.

But perhaps I missed something, which is why I'm asking.

Reply:
I also checked #7318 where these were introduced, and it looks like these were the patterns right from the start, so it doesn't look like there were added later.

Reply:
Probably used values from user feedback for the examples. So I have no particular opinion.